Makefile

```diff
@@ -20,16 +18,20 @@ tidy:
 	$(GO_BIN) mod tidy
 .PHONY: tidy
 
-lint: .nomaprange.so
-	$(GOLINT) run ./...
-.PHONY: lint
-
-.nomaprange.so:
-	$(eval TMP=$(shell mktemp -d))
-	(cd $(TMP); go mod init a; go build -buildmode=plugin -o .nomaprange.so github.com/Kubuxu/go-no-map-range/plugin)
-	cp $(TMP)/.nomaprange.so .nomaprange.so
-	rm -rf $(TMP)
-
 gen:
 	$(GO_BIN) run ./gen/gen.go
 .PHONY: gen
+
+
+# tools
+toolspath:=support/tools
+
+$(toolspath)/golangci-lint: $(toolspath)/go.mod
+	(cd $(toolspath); go build github.com/golangci/golangci-lint/cmd/golangci-lint)
```

Comment:
```suggestion
	(cd $(toolspath); go build -o "$@" github.com/golangci/golangci-lint/cmd/golangci-lint)
```

Reply:
(just to be safe)

Reply:
`$@` is relative, and we have to `cd` for go.mod to work

Reply:
Did this: `$(@:$(toolspath)/%=%)`